README.md

```diff
@@ -446,6 +440,8 @@ Puppet Labs modules on the Puppet Forge are open projects, and community contrib
 
 We want to keep it as easy as possible to contribute changes so that our modules work in your environment. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
 
-You can read the complete module contribution guide [on the Puppet Labs wiki.](http://projects.puppetlabs.com/projects/module-site/wiki/Module_contributing)
+For more information, see our [module contribution guide.](https://docs.puppetlabs.com/forge/contributing.html)
+
+###Contributors
 
-Licensed under [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)
```

Comment:
Makes sense not to repeat yourself, but possibly a link then to the license? I know in some cases other managers are asking for it to be spelled out in the readme, so I'd like to get consistency here. cc @stahnma